Mjml.Net/Components/Body/DividerComponent.cs

```diff
@@ -0,0 +1,118 @@
+                ["padding"] = AttributeTypes.FourPixelsOrPercent,
+                ["padding-bottom"] = AttributeTypes.PixelsOrPercent,
+                ["padding-left"] = AttributeTypes.PixelsOrPercent,
+                ["padding-right"] = AttributeTypes.PixelsOrPercent,
+                ["padding-top"] = AttributeTypes.PixelsOrPercent,
+                ["width"] = AttributeTypes.PixelsOrPercent,
+            };
+
+        public override Attributes? DefaultAttributes { get; } =
+            new Attributes
+            {
+                ["align"] = "center",
+                ["border-color"] = "#000000",
+                ["border-style"] = "solid",
+                ["border-width"] = "4px",
+                ["padding"] = "10px 25px",
+                ["width"] = "100%",
+            };
+
+        public override void Render(IHtmlRenderer renderer, INode node)
+        {
+            var borderStyle = $"{node.GetAttribute("border-width")} {node.GetAttribute("border-style")} {node.GetAttribute("border-color")}";
+
+            RenderDefault(renderer, node, borderStyle);
+            RenderOutlook(renderer, node, borderStyle);
+        }
+
+        private static void RenderDefault(IHtmlRenderer renderer, INode node, string borderStyle)
+        {
+            renderer.ElementStart("p")
+                .Style("font-size", "1px")
+                .Style("border-top", borderStyle)
+                .Style("width", node.GetAttribute("width"));
+
+            renderer.ElementEnd("p");
+        }
+
+        private static void RenderOutlook(IHtmlRenderer renderer, INode node, string borderStyle)
+        {
+            var outlookWidth = GetOutlookWidth(renderer, node);
+
+            renderer.Content("<!--[if mso | IE]>");
+
+            renderer.ElementStart("table")
+                .Attr("align", GetAlign(node))
+                .Attr("border", "0")
+                .Attr("cellpadding", "0")
+                .Attr("cellpadding", "0")
+                .Attr("role", "presentation")
+                .Attr("width", outlookWidth)
+                .Style("font-size", "1px")
+                .Style("border-top", borderStyle)
+                .Style("width", node.GetAttribute("width"));
+
+            renderer.ElementStart("tr");
+
+            renderer.ElementStart("td")
+                .Attr("style", "height:0; line-height:0;");
+
+            renderer.Content("&nbsp");
+
+            renderer.ElementEnd("td");
+            renderer.ElementEnd("tr");
+
+            renderer.Content("<![endif]-->");
+        }
+
+        private static string GetAlign(INode node)
+        {
+            switch (node.GetAttribute("align"))
+            {
+                case "left":
+                    return "0px";
+                case "right":
+                    return "0px 0px 0px auto";
+                default:
+                    return "0px auto";
+            }
+        }
+
+        private static string GetOutlookWidth(IHtmlRenderer renderer, INode node)
```

Comment:
From memory, this function appears in multiple components. I think we should move this into a NodeHelper possibly.